src/main.rs

```diff
@@ -119,20 +119,21 @@ struct App {
     )]
     prometheus_port: u16,
     #[structopt(
-        long = "max_rates_saved",
+        long = "max-rates-saved",
         help = "Determines the size of the history of rates from the exchange",
         env = "EUR2CCD_SERVICE_MAX_RATES_SAVED",
         default_value = "60"
     )]
     max_rates_saved: usize,
     #[structopt(
-        long = "test-exchange",
-        help = "If set to true, pulls exchange rate from the given location (see local_exchange \
-                subproject)  (FOR TESTING)",
-        env = "EUR2CCD_SERVICE_TEST_EXCHANGE",
+        long = "test-source",
+        help = "If set to true, pulls exchange rate from each of the given locations (see \
+                local_exchange subproject)  (FOR TESTING)",
+        env = "EUR2CCD_SERVICE_TEST_SOURCE",
+        use_delimiter = true,
         group = "testing"
     )]
-    test_exchange: Option<Url>,
+    test_source: Option<Vec<Url>>,
```

Comment:
I would recommend using just `Vec<Url>` here and naming it `test_sources`.

Reply:
Don't we need to also do `use_delimiter = true` or is that done by `structopt`? In any case it would be nice to be explicit.

Reply:
I've changed the the name, and remove the Option. (But use_delimiter is already there, I don't think it allows it implicitly)